CONTRIBUTING.md

```diff
@@ -1,78 +1,116 @@
-# Contributing Guidelines
-If you're interested in contributing to this project, here are a few ways to do so:
+# Contribution Guidelines
 
-**Note:** If these contributing guidelines are not followed we will close the issue or PR, please
-read these instructions carefully.
+**Note:** If these contribution guidelines are not followed your issue or PR might be closed, so
+please read these instructions carefully.
 
-## Bug fixes
+## Contribution types
+
+### Bug fixes
```

Comment:
"Bug fix" is a PR that fixes an existing bug. This section is more about bug issues. So, maybe "Bug reports"?

Reply:
Split it up in two sections, wdyt?